subprojects/platform-base/src/integTest/groovy/org/gradle/language/base/LanguageTypeSampleIntegrationTest.groovy

```diff
@@ -27,15 +27,13 @@ class LanguageTypeSampleIntegrationTest extends AbstractIntegrationSpec {
     @Rule
     Sample languageTypeSample = new Sample(temporaryFolder, "customModel/languageType")
 
-    def setup() {
-        executer.usingInitScript(RepoScriptBlockUtil.createMirrorInitScript())
```

Comment:
Why doesn't this need the init script any more?

Reply:
This sample's `buildSrc/build.gradle` contains `jcenter()`, which can't be updated by init script.  But I should use init script, at least we can make the main build benefit. I'll add it back. https://github.com/gradle/gradle-private/issues/1403